services/WorkflowService.ts

```
@@ -0,0 +1,174 @@
+      throw error;
+    }
+  }
+
+  static async updateWorkflow(
+    workflowId: string,
+    workflow: UpdateWorkflowRequest
+  ): Promise<Workflow | undefined> {
+    try {
+      const headers = await getHeaders();
+      const response = await axios.put(
+        `${this.BASE_URL}/${workflowId}`,
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)